tests/ingresstests/l7_graph_test.go

```diff
@@ -2782,3 +2782,184 @@ func TestPortsForInsecureAndSecureSNI(t *testing.T) {
 
 	TearDownTestForIngress(t, modelName)
 }
+
+func TestV6BackendService(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+	modelName := "admin/cluster--Shared-L7-0"
+	objects.SharedAviGraphLister().Delete(modelName)
+
+	v6Svc := integrationtest.ConstructService("default", "avisvc", corev1.ProtocolTCP, corev1.ServiceTypeClusterIP, false, make(map[string]string))
+	ipFamilyPolicy := corev1.IPFamilyPolicy("SingleStack")
+	v6Svc.Spec.IPFamilies = []corev1.IPFamily{"IPv6"}
+	v6Svc.Spec.IPFamilyPolicy = &ipFamilyPolicy
+	_, err := KubeClient.CoreV1().Services("default").Create(context.TODO(), v6Svc, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatalf("error in adding Service: %v", err)
+	}
+	integrationtest.CreateEP(t, "default", "avisvc", false, false, "ff06::")
+
+	ingrFake1 := (integrationtest.FakeIngress{
+		Name:        "ingress-v6-backend-svc",
+		Namespace:   "default",
+		DnsNames:    []string{"foo.com"},
+		Paths:       []string{"/foo"},
+		ServiceName: "avisvc",
+	}).Ingress()
+	_, err = KubeClient.NetworkingV1().Ingresses("default").Create(context.TODO(), ingrFake1, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatalf("error in adding Ingress: %v", err)
+	}
+	integrationtest.PollForCompletion(t, modelName, 5)
+	found, aviModel := objects.SharedAviGraphLister().Get(modelName)
+	if found {
+		nodes := aviModel.(*avinodes.AviObjectGraph).GetAviVS()
+		g.Expect(*nodes[0].PoolRefs[0].Servers[0].Ip.Addr).To(gomega.Equal("ff06::1"))
+
+	} else {
+		t.Fatalf("Could not find model: %s", modelName)
+	}
+
+	err = KubeClient.NetworkingV1().Ingresses("default").Delete(context.TODO(), "ingress-v6-backend-svc", metav1.DeleteOptions{})
+	if err != nil {
+		t.Fatalf("Couldn't DELETE the Ingress %v", err)
+	}
+	integrationtest.DelSVC(t, "default", "avisvc")
+	integrationtest.DelEP(t, "default", "avisvc")
+	VerifyIngressDeletion(t, g, aviModel, 0)
+
+	TearDownTestForIngress(t, modelName)
+}
+
+func TestDualStackBackendService(t *testing.T) {
```

Comment:
@aaha97: Is it possible to simulate a test case where one pool member is IPV4 and other is IPV6?

Reply:
this case is not required